specification/dimensions/commitmentid.md

```diff
@@ -0,0 +1,32 @@
+# Commitment ID
+
+A commitment-based discount is a contractual commitment to use a certain amount of a specific type of usage for a fixed term in exchange for a discounted unit price. Commitment-based discounts can cover either usage or cost of various resources such as virtual machines or databases. A provider can offer various pricing models, payment options, and term durations for commitment-based discounts.
```

Comment:
I really like this paragraph, but it is nothing specifically to do with the ID. ie. it applies to all Commitment fields - it just happens to be on the top of this one to give it context. Does this mean all Commitment Discount fields should have it? Or should it be abstracted into a base class?

Reply:
There's [this](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/glossary.md) file. It's empty though.

Reply:
Pretty much patterned with BillingAccount[Id|Name], Resource[Id|Name], and SubAccount[Id|Name] columns.

Reply:
We'll mention this to Karl as part of the glossary effort! We'll clean this up as needed once we have the right guidance. Closing this for now.